packages/api/src/router/activity.ts

```diff
@@ -0,0 +1,95 @@
+import { z } from "zod";
+
+import { adminProcedure, createTRPCRouter } from "../trpc";
+
+export const activityRouter = createTRPCRouter({
+  getActivity: adminProcedure
+    .input(z.object({ id: z.number().nonnegative().optional() }))
+    .query(async ({ ctx, input }) => {
+      return input.id
+        ? await ctx.db.activity.findFirst({
+            where: { id: input.id },
+          })
+        : await ctx.db.activity.findMany({
+            orderBy: { day: "desc" },
+          });
+    }),
+  getSchedule: adminProcedure
+    .input(z.object({ day: z.string() }))
+    .query(async ({ ctx, input }) => {
```

Comment:
this can be a `protectedProcedure` since the users of the app will also be able to see the schedule